pkg/coscheduling/coscheduling_test.go

```diff
@@ -39,6 +39,11 @@ import (
 	testutil "sigs.k8s.io/scheduler-plugins/test/util"
 )
 
+func createPodInfo(pod *v1.Pod) *framework.PodInfo {
```

Comment:
nit: move it to a shared place and renamed to `func mustNewPodInfo(t *testing.T, pod *v1.Pod) *framework.PodInfo {`, so we can fatalf the non-nil error.

Reply:
done!